modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/AppDatabase.kt

```diff
@@ -989,6 +997,45 @@ abstract class AppDatabase : RoomDatabase() {
             database.execSQL("ALTER TABLE podcasts ADD COLUMN funding_url TEXT")
         }
 
+        val MIGRATION_113_114 = addMigration(113, 114) { database ->
+            with(database) {
+                beginTransaction()
+                try {
+                    database.execSQL(
```

Comment:
```suggestion
                    execSQL(
```
I could be wrong, but since you have used the `with` statement, you don't need to use a database variable, right?